Sources/SwiftDocC/Model/Rendering/RenderNodeTranslator.swift

```diff
@@ -736,11 +736,27 @@ public struct RenderNodeTranslator: SemanticVisitor {
         node.topicSectionsStyle = topicsSectionStyle(for: documentationNode)
         
         if shouldCreateAutomaticRoleHeading(for: documentationNode) {
-            if node.topicSections.isEmpty {
-                // Set an eyebrow for articles
+            
+            let role = DocumentationContentRenderer.roleForArticle(article, nodeKind: documentationNode.kind)
+            node.metadata.role = role.rawValue
+            
+            switch role {
+            // If there are no links to other nodes from the article,
+            // set the eyebrow for articles.
+            case .article:
                 node.metadata.roleHeading = "Article"
+            // If the article links to other nodes, set the eyebrow for
+            // API Collections if any linked node is a symbol.
+            case .collectionGroup:
+                let curatesSymbols = contentCompiler.collectedTopicReferences.contains { topicReference in
+                    context.topicGraph.nodeWithReference(topicReference)?.kind.isSymbol ?? false
+                }
+                if curatesSymbols {
```

Comment:
Can you add an `else` clause and explicitly set `node.metadata.roleHeading = ""`, along with a comment explaining what you mentioned in the PR description?

> If the entire list does not contain a symbol (i.e., it's a plain collection), we maintain the old behavior of not displaying anything as the eyebrow title.


You were talking about when  `curatesSymbols` is false correct?

Otherwise to me this reads like a bug and I'm left wondering what the role heading should be when there are no symbols.

Reply:
@patshaughnessy I'm not sure if adding the unnecessary set `node.metadata.roleHeading = ""` is the best option here, instead I updated the comments to the following:
```swift
case .collectionGroup:
    // If the article links to other nodes, set the eyebrow for
    // API Collections if any linked node is a symbol.
    //
    // If none of the linked nodes are symbols (it's a plain collection),
    // don't display anything as the eyebrow title.
    let curatesSymbols = contentCompiler.collectedTopicReferences.contains { topicReference in
        context.topicGraph.nodeWithReference(topicReference)?.kind.isSymbol ?? false
    }
    if curatesSymbols {
        node.metadata.roleHeading = "API Collection"
    }
```

Reply:
>Otherwise to me this reads like a bug and I'm left wondering what the role heading should be when there are no symbols.

Since the default case below also doesn't set a role heading I feel that this code reads fine. 

The main alternative I can think of that would more clearly indicate that this case is only meant to cover a subset of collection groups would be to move the `contains(where:)` check into a `where` clause on the case:

```swift
case .collectionGroup where contentCompiler.collectedTopicReferences.contains(where: { context.topicGraph.nodeWithReference($0)?.kind.isSymbol == true }):
    node.metadata.roleHeading = "API collection"
```

However, I feel like this loses some of the descriptiveness of what the where clause is checking that the `curatesSymbols` variable provides. A comment could possibly make up for some of that by describing what the where clause is checking for but it's still a readability compromise:

```swift
case .collectionGroup where contentCompiler.collectedTopicReferences.contains(where: { context.topicGraph.nodeWithReference($0)?.kind.isSymbol == true }):
    // Collection groups that curate symbols are marked as _API_ collections.
    node.metadata.roleHeading = "API collection"
```

Reply:
Clever - I never think of using `where` on switch cases or for loops. I think Sofia's new comment makes it more clear what is happening here. Thanks!